examples/opcuaDeviceShifu/server/generate_cert.go

```diff
@@ -90,7 +90,7 @@ func main() {
 		log.Fatalf("error closing %s: %s", certFile, err)
 	}
 
-	log.Printf("wrote %s\n", certFile)
+	log.Printf("wrote %s", certFile)
```

Comment:
let's keep things consistent, for all logging places, let's use klog.

Reply:
this file not run in deviceshifu,  just to generate certification for opcua, user can use `go run generate_cert.go
` directly

Reply:
Yeah, but they are still in the same repo and share the same go.mod, so the logging should be consistent.

Reply:
this file just a tool for generate certificate, it shouldn't rely on go.mod, using klog could make things more complicated.

Reply:
use fmt.Print instead then.